.github/workflows/ci.yaml

```diff
@@ -8,9 +8,27 @@ jobs:
     name: Test Suite
     runs-on: ubuntu-latest
     steps:
-      - name: Install llvm-config
-        run: sudo apt-get install llvm
       - uses: actions/checkout@v2
+      - name: Install compiler tools
+        run: sudo apt-get install llvm automake libxml2-dev gperf build-essential libboost-all-dev
+      - name: Checkout UDBM
+        uses: actions/checkout@v2
+        with:
+          repository: 'UPPAALModelChecker/UDBM'
+          ref: '26482eb64e74d75be5ee647587a35549d453bf43'
+          path: 'udbm'
+      - name: Compile UDBM
+        run: |
+          cd udbm
+          ./autogen.sh
+          ./configure
+          make
+          ./scripts/mergelibs.sh
+          cp ./lib/libudbm.a ../dbm/libudbm.a

```

Comment:
Im having problems compiling the code, because im missing the `libudbm.a` file. Currently i cant find anything on getting this file for compilation. Am i missing something or am i supposed to follow CI setup?

Reply:
I will add some instructions on how to build a copy of the UDBM library